### PR TITLE
closes #6

### LINKED ISSUE #6: Review Model
# Properties

- [ ] Stars 1-5 _required_ I don't think this should be type: Integer, I believe there is a more specific type
- [ ] Description _required probably? text_

# Relationships

None! Belongs to a bunch of stuff tho

### PR DESCRIPTION
**Pull request by melmquist**
Made review model with requiring an integer between 1-5, maybe want to change to allow for ratings like 2.5?

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -7,3 +7,5 @@ var User = require('./models/user');
 
 // if we had more models, we could associate them in this file
 // e.g. User.hasMany(Reports)
+
+Review.belongsTo(User);
```
Comment (kaptiin-tuekhov) [inline server/db/index.js]: Do we need both Review belongs to User and User has many Reviews?

Comment (Babbins) [inline server/db/index.js]: bleongsTo creates a one-to-one relationship. we want User.hasMany(Review, {as: 'reviews'});

**Comment by pimeson** [comment]
I don't think we need to do half ratings. I like how amazon does it with hover over star ratings.

File: server/db/models/review.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+var crypto = require('crypto');
+var _ = require('lodash');
```
Comment (pimeson) [inline server/db/models/review.js]: Crypto and lodash are unnecessary requires.

**Pull request merged**